website/docs/reference/project-configs/packages-install-path.md

```diff
@@ -9,11 +9,6 @@ default_value: dbt_packages
 
 </Changelog>
 
-<File name='dbt_project.yml'>
-
-```yml
-packages-install-path: directorypath
-```
 
 </File>
```

Comment:
Hello! We need to verify this information before approving, but in the meantime, I noticed the build is failing because we'd need to delete the closing file tag.
```suggestion
```

Reply:
Understandable!